### PR TITLE
feat: add record-context

### PR DESCRIPTION
**Pull request by AdrianValdes**

### CODE REVIEW

File: src/helpers/apiCalls.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { LOGIN_USER, SIGNUP_USER } from '../context/constants';
+import { FETCH_RECORDS, LOGIN_USER, SIGNUP_USER } from '../context/constants';
```
Comment (chmette) [inline src/helpers/apiCalls.js:1]: is it best practice to write it in capitals?

File: src/records/Records.jsx [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
-);
+export const Records = () => {
+  const { records } = useRecords();
+  return (
+    <GridStyles>
+      <PrimaryText weight='bold'>Dashboard</PrimaryText>
+      <PrimaryText>Here you can find all your records.</PrimaryText>
+      <GridRecords>
+        {records.map((record) => (
+          <div key={record.title}>{record.title}</div>
```
Comment (chmette) [inline src/records/Records.jsx:14]: think its just for testing but here just need to be the cover.

File: src/helpers/apiCalls.js (hunk 1 above)
Comment (AdrianValdes) [reply src/helpers/apiCalls.js:1]: It is convention for the type of the actions for useReducer, it comes from something `redux` started and useReducer is based on that.

File: src/records/Records.jsx (hunk 2 above)
Comment (AdrianValdes) [reply src/records/Records.jsx:14]: I wanted to create a little box with the cover and the title. I have the cover but there is no image and it looks weird.

File: src/helpers/apiCalls.js (hunk 1 above)
Comment (chmette) [reply src/helpers/apiCalls.js:1]: ok, danke

File: src/records/Records.jsx (hunk 2 above)
Comment (chmette) [reply src/records/Records.jsx:14]: yes, teh covers are not final yet

**Review by chmette — APPROVED** [review]
thanks

File: src/records/Records.jsx (hunk 2 above)
Comment (AdrianValdes) [reply src/records/Records.jsx:14]: I saw you created a shakira record :D But the cover is coming from the local. I remember Vasilis couldn't solve the path thing, do you know how to solve it?

**Pull request merged**